azure-pipelines.yml

```diff
@@ -17,6 +17,12 @@ variables:
   buildConfiguration: 'Release'
 
 steps:
+- task: UseDotNet@2
```

Comment:
Shouldn't this say `UseDotNet@3`?

Reply:
It's actually the version of the specified Azure Pipelines Task. [Here](https://github.com/microsoft/azure-pipelines-tasks/tree/master/Tasks) you can find kind of a list of available Tasks and versions.